examples/blueprint-construct/index.ts

```diff
@@ -105,7 +105,7 @@ export default class BlueprintConstruct {
             new blueprints.addons.KarpenterAddOn({
                 requirements: [
                     { key: 'node.kubernetes.io/instance-type', op: 'In', vals: ['m5.2xlarge'] },
-                    { key: 'topology.kubernetes.io/zone', op: 'NotIn', vals: ['us-west-2c']},
+                    { key: 'topology.kubernetes.io/zone', op: 'In', vals: ['us-west-2a','us-west-2b','us-west-2c']},
```

Comment:
Intentional change to include all azs in karpenter?

Reply:
changed from local testing. Will restore.